pages.de/common/ansible.md

```diff
@@ -0,0 +1,29 @@
+# ansible
```

Comment:
Hey @ProfileID you have submitted `ansible` in #4403 please remove the page from this PR